src/Players/HumanPlayer.hs

```diff
@@ -49,6 +50,13 @@ validMoveString str = False
 
 -- Converts a chess move from the "[a-h][1-8] to [a-h][1-8]" format to a ChessMove
 -- WARNING: Assumes that the string is in a valid format
--- TODO: Implement + test this (1.5 hour) [Cynthia]
 stringToChessMove :: String -> ChessMove
-stringToChessMove _ = ChessMove ('A', 0) ('A', 0)
\ No newline at end of file
+stringToChessMove string = 
+    ChessMove (stringToChessMoveHelper fromPosition) (stringToChessMoveHelper toPosition)
+    where
+        fromPosition = take 2 [ x | x <- string, x /=' ', x /='t', x/='o']
+        toPosition = drop 2 [ x | x <- string, x /=' ', x /='t', x/='o'] 
+
+stringToChessMoveHelper :: [Char] -> (Char, Integer)
```

Comment:
you could probably rename `stringToChessMoveHelper` to `readChessPosition`. Alternatively, if you added this as a Read instance function to ChessPosition then you'll just be able to do `read fromPosition` and `read toPosition` where:

```haskell
instance Read ChessPosition where
    read position = (string !! 0, toInteger (digitToInt (string !! 1)))
```